### ISSUE TITLE
cifs mount not shown, what to do?

### ISSUE THREAD

**Issue by JapCLoN** [cockpit-project/cockpit#16460]
Cockpit version: 188
OS: Raspian lite
Page: Storage

I have a mount on my server running cockpit, but it is now shown in the storage tap?

**Comment by fahrradflucht** [comment]
From my understanding, this "issue" is actually a missing feature. SMB/CIFS mounts are ignored by the storage tab. I would also welcome this addition very much.

**Comment by martinpitt** [comment]
Right, the Storage page manages physical/local storage devices (what /dev and udisks2 see). NFS is indeed some outlier there, and quite frankly it shouldn't really have been there. But https://github.com/45Drives/cockpit-file-sharing is a Cockpit UI for SMB/CIFS, maybe that works for you?

**Comment by tarmacalastair** [comment]
Just been looking at my cockpit to check if my machine could see the samba shares I've recently added and noticed the same problem.
This issue has been closed, referring to _cockpit-file-sharing_ instead but this plugin is for serving up samba shares, not for viewing samba mounts **from a remote server** that you might want to see on your cockpit screen.

So it looks like the only way to see the samba shares you are mounting is to examine _/etc/fstab_

It would be great to see this feature added. Or if the storage page is not supposed to show NFS mounts then maybe a new plugin to show remote storage (CIFS **and** NFS) that are accessible **to** the machine that you are currently viewing (rather then shares being server up by this machine).

Thanks
